src/town/town_manager.py

```diff
@@ -40,7 +43,8 @@ def wait_for_town_spawn(self, time_out: float = None) -> Location:
         template_match = self._template_finder.search_and_wait([
             "A5_TOWN_0", "A5_TOWN_1",
             "A4_TOWN_4", "A4_TOWN_5",
-            "A3_TOWN_0", "A3_TOWN_1"
+            "A3_TOWN_0", "A3_TOWN_1",
+            "A2_TOWN_LIGHT_WELL", "A2_TOWN_MEDIUM_WELL", "A2_TOWN_DARK_WELL"
```

Comment:
Is it all the same template just on different daytimes? That is a bit tricky. Cause it can happen that your merc is in front of that template. That is why all the other town templates are from 2 different locations.

Reply:
Its a really odd situation mercenary in front of the well. Even so, wait_for_town_spawn() has a high timeout value enought to mercenary leave that area